persistence-modules/spring-data-jpa-enterprise/src/test/java/com/baeldung/SpringContextTest.java

```diff
@@ -3,12 +3,14 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit4.SpringRunner;
 
 import com.baeldung.boot.Application;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = Application.class)
+@TestPropertySource(properties = {"spring.jpa.show-sql=false "})
```

Comment:
Maybe better to set this property in the application(-test).properties?

Reply:
Yes sure.